### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by ksenia-krasheninnikova**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/genomeassembly/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by ksenia-krasheninnikova** [comment]
@gq1 @muffato The github CI checks are fine, I'm happy to merge as long as get an approval. Thanks :)

**Comment by gq1** [comment]
The CI job only submit the job on farm, not sure why pending so long. This is the first time you run all the code together on dev, better to wait.

**Review by gq1 — APPROVED** [review]

**Pull request merged**
